build.sbt

```diff
@@ -0,0 +1,234 @@
+import UnidocKeys._
+//
+
+lazy val V = new {
+  lazy val cats                = "0.8.1"
```

Comment:
We should get this updated to the latest cats version. Soon in sbt-catalyst-extra @juanpedromoreno and I are planning on integrating @fedefernandez plugin to autocheck dependencies so libs don't fall behind. Stay tunned for more on this, we will ask all 47deg libs to update to that when ready.